src/Common/tests/System/Diagnostics/DebuggerAttributes.cs

```diff
@@ -0,0 +1,140 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+
+namespace System.Diagnostics
+{
+    internal static class DebuggerAttributes
+    {
+        internal static object GetFieldValue(object obj, string fieldName)
```

Comment:
That's fine.  We can disable them when running the tests on .NET Native.  (That said, if this test fails, so too would debugging in the IDE, unless the test and F5 run against different assemblies... I realize that's possible.)